### PR TITLE
1195 - icon when manually added

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/entities/trialSessions/TrialSession.js [hunk 1]
```diff
@@ -370,4 +384,22 @@ TrialSession.prototype.removeCaseFromCalendar = function({
   return this;
 };
 
+/**
+ * removes the case totatlly from the trial session
+ *
+ * @param {object} arguments the arguments object
+ * @param {string} arguments.caseId the id of the case to remove from the calendar
+ * @param {string} arguments.disposition the reason the case is being removed from the calendar
```
Comment (rachaelparris) [inline shared/src/business/entities/trialSessions/TrialSession.js]: Jsdoc typo `totatlly` and can remove disposition from the args.

File: web-client/src/views/TrialSessionDetail/AllCases.jsx [hunk 2]
```diff
@@ -33,6 +35,12 @@ export const AllCases = connect(
                 <td>
                   <CaseLink formattedCase={item} />
                 </td>
+
+                <td>
+                  {item.isManuallyAdded && (
+                    <FontAwesomeIcon icon="calendar-plus" />
```
Comment (rachaelparris) [inline web-client/src/views/TrialSessionDetail/AllCases.jsx]: `className="mini-success"` should be all you need to make this icon green.

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/TrialSessionDetail/AllCases.jsx [hunk 3]
```diff
@@ -21,6 +22,7 @@ export const AllCases = connect(
           <thead>
             <tr>
               <th>Docket</th>
+              <th></th>
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionDetail/AllCases.jsx]: If this column can convey any meaning to the user, please provide an `aria-label` describing it.  if not, set `aria-hidden=true`.  Same for the icons which may or may not appear in the table itself.

**Review by kkoskelin — CHANGES_REQUESTED** [review]
See comment about the new icon column.  Which ever decision you make, please apply to both views.  Thanks!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**